packages/core/test/popover/popoverTests.tsx

```diff
@@ -290,18 +346,6 @@ describe("<Popover>", () => {
                     autoFocus: false,
                 });
             });
-
-            it("popover remains open after target focus if autoFocus={true}", () => {
```

Comment:
this test doesn't really make sense since `autoFocus` is disabled for "hover" popovers. it should be removed. see code:

https://github.com/palantir/blueprint/blob/c3c1a47b8717cbcbb06d9ac3bd3cdad67ddb1a4f/packages/core/src/components/popover/popover.tsx#L483